subjects/descendappendrange/README.md

```diff
@@ -0,0 +1,46 @@
+## descendappendrange
+
+### Instructions
+
+- Write a function that takes an `int` max and an `int` min as parameters. The function should return a slice of `int`s with all the values between the max and min.
+
+- The `Max`must be included, and min must be excluded.
+
+- If max is inferior than or equal to min, a `nil` slice is returned.
```

Comment:
Nil slice does not exist. Either have "Empty slice" or return a `nil` value. 
"If max is inferior than or equal to min, (an empty slice)/(a `nil` value) is returned.